src/applications/simple-forms-form-engine/shared/config/pages/employmentHistory.js

```diff
@@ -69,24 +69,6 @@ export const detailPage = {
   },
 };
 
-/** @returns {PageSchema} */
-export const introPage = options => ({
-  path: 'employers',
-  title: 'Employers',
-  uiSchema: {
-    ...webComponentPatterns.titleUI(
-      `Treatment records`,
-      `In the next few questions, we’ll ask you about the treatment records you’re requesting. You must add at least one treatment request. You may add up to ${
-        options.maxItems
-      }.`,
-    ),
-  },
-  schema: {
-    type: 'object',
-    properties: {},
-  },
-});
-
```

Comment:
The Employment History variation is optional and will never need an introduction page, making this code irrelevant.